app/src/main/res/values/strings.xml

```diff
@@ -44,4 +44,9 @@
     <string name="book_language">Book language</string>
     <string name="your_picture_of_your_book_condition">Your picture of your book condition</string>
     <string name="email">Email</string>
+    <string name="textview">TextView</string>
+    <string name="user_profile">user_profile</string>

```

Comment:
`<string name="user_picture">user_picture</string>`
might be a better name? I feel like the profile is the entire page